blockchains/ethereum/assets/0xbAe226690D7DCe1Da1823f0BF811529b8229134a/info.json

```diff
@@ -0,0 +1,6 @@
+{
+    "name": "IMO",
+    "website": "https://www.imoex.top",
```

Comment:
can you present information from `imoex.top` and link to official token address?

Reply:
I have present pull request and token information on **imoex.top** . 
[https://www.imoex.top/en_US/noticeInfo/10280](https://www.imoex.top/en_US/noticeInfo/10280)